rules/signing.bzl

```diff
@@ -339,36 +340,48 @@ offline_signature_attach = rule(
     },
 )
 
-def _sign_bin_impl(ctx):
-    rsa_key = key_from_dict(ctx.attr.rsa_key, "rsa_key")
-    spx_key = key_from_dict(ctx.attr.spx_key, "spx_key")
+def sign_binary(ctx, **kwargs):
```

Comment:
Maybe document the content of the returned dictionary?

Reply:
Done.